.golangci.yml

```diff
@@ -74,14 +72,14 @@ linters:
     - predeclared
     - promlinter
     - reassign
-    - revive
+    # - revive
```

Comment:
Could you add a small comment just to mention why (especially this one since I believe it was the one with the storeId -> storeID lint?)

Reply:
I wanted to do this in a separate PR as highest risk of breakage